collect.py

```diff
@@ -71,43 +75,67 @@ def store_results(temppath):
     print('Results stored in %s.' % fname)
 
 
+def parse_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-m', '--model',
+        help='Model to operate in. Accepts <controller name>:<model name>')
+    return parser.parse_args()
+
+
 def main():
+    options = parse_args()
+    model = options.model
+    if model:
+        if ':' in options.model and len(options.model.split(':')) == 2:
+            model = "-m {}".format(options.model)
```

Comment:
We need else cases for both of these, yeah?

If no model is specified by the user, then we either need to grab the current model with `juju switch`, or we need to not pass the model through to any of the juju commands.

If the user provides a model with no `:` in it, then we still need to add the `-m` to the front for the rest of the script to work.